Graphics/GLSLTools/src/SPIRVUtils.cpp

```diff
@@ -551,5 +565,166 @@ std::vector<unsigned int> GLSLtoSPIRV(const SHADER_TYPE ShaderType, const char*
         return std::move(SPIRV);
     }
 }
+} // namespace Diligent
+
+// This is modified version of StripReflectInfoPass from SPIRV-Tools
```

Comment:
Why is this necessary?

Reply:
Because DXC adds SPV_GOOGLE_user_type that doesn't deleted by origin strip-reflection pass and causes validation errors